### PR TITLE
Fix confirm button still available after tx confirmation

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1346

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt [hunk 1]
```diff
@@ -197,7 +197,7 @@ class TransactionDetailsViewModel
                 var rejectable = false
                 var safeOwner = false
                 if (executionInfo is DetailedExecutionInfo.MultisigExecutionDetails) {
-                    awaitingConfirm = isAwaitingOwnerConfirmation(executionInfo, txDetails!!.txStatus, owners)
+                    awaitingConfirm = false
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt:200]: Why is it always false and why is isAwaitingOwnerConfirmation() not working correctly here?

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt:200]: because we just submitted a confirmation

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt:200]: we haven't updated our txDetails and that was the reason isAwaitingOwnerConfirmation was not working correctly

**Review by biafra23 — APPROVED** [review]

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt [hunk 2]
```diff
@@ -190,6 +190,7 @@ class TransactionDetailsViewModel
                     credentialsRepository.signWithOwner(owners[0], executionInfo.safeTxHash.hexToByteArray())
                 )
             }.onSuccess {
+                txDetails = it
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsViewModel.kt:193]: This doesn't seem to be enough. The button is still there until I pull to refresh.

**Pull request merged**